examples/drag-and-drop-custom.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: example.html
+title: Custom Drag-and-Drop
+shortdesc: Example of using the drag-and-drop interaction with custom formats.
+docs: >
+  Example of using the drag-and-drop interaction with custom formats.
+  In addition to the formats used in the [Drag-and-Drop](drag-and-drop.html) example a custom format (subclassing KML) is used to handle KMZ files, the standard KML format is pre-constructed to not extract styles, and individual MVT tiles can be previewed.
+  There is no projection transform support, so this will only work with data in EPSG:4326 and EPSG:3857.
```

Comment:
There is a lot of nice stuff in this example.  Rather than one example that does three things (read KMZ, read KML without styles, view MVT tiles), I think it would be simpler to have three examples that demonstrate one thing each.

This would avoid users being confused about something like the MVT tile coordinate inputs when viewing KML, for example.